### PR TITLE
Add remorse popup for closing all tabs

### PR DESCRIPTION
**Pull request by rainemak**
This contains two commits.

### CODE REVIEW

**Comment by monich** [comment]
That's a bit of an overkill :) With the remorse timer, the setting is unnecessary, IMO.

**Comment by rainemak** [comment]
@monich I know that some people likes to remove "Close all tabs" completely. This would satisfy that need as well.

**Comment by rainemak** [comment]
Updated PR and cleaned settings side of it

**Comment by pvuorela** [comment]
LGTM

File: src/pages/components/TabView.qml [hunk 1]
```diff
@@ -77,6 +77,11 @@ SilicaListView {
         defaultValue: true
     }
 
+    // Remorse popup for closing all tabs.
```
Comment (pvuorela) [inline src/pages/components/TabView.qml]: Element and id say pretty much the same thing already :)

**Pull request merged**